firmware/Makefile

```diff
@@ -33,7 +33,7 @@ deb: all
 $(OSX_DRV):
 	@echo ""
 	@echo "Checking dependencies for driver download..."
-	@which curl xzcat cpio
+	@if [ 3 != "$(which curl cpio xzcat| wc -l)" ]; then echo -e "Missing commands, please check if your OS has curl, cpio and xzcat.\n" && exit 1; fi
```

Comment:
Might be cleaner as:

`@for required in curl cpio xzcat; do if which $required > /dev/null; then echo "Found $required"; else echo "Missing $required. Exiting."; exit 1; fi; done`